api-specs/gem.proto

```diff
@@ -62,3 +64,12 @@ message GetResponse {
 message GetPendingCollectionForUserResponse {
   repeated Gem gems = 1;
 }
+
+message GemLogs {
+  map<int64, GemLogsWithFriend> gem_logs = 1;
+}
+
+message GemLogsWithFriend {
+  usersapi.User friend = 1;
```

Comment:
I'm thinking of modifying the Gem protobuf so that instead of storing creator_id and receiver_id, it stores creator and receiver with User type instead.  Would that be desirable?